src/Microsoft.DotNet.Darc/src/DarcLib/Helpers/HttpRequestManager.cs

```diff
@@ -50,7 +50,7 @@ public async Task<HttpResponseMessage> ExecuteAsync(int retryCount = 15)
                     {
                         if (!string.IsNullOrEmpty(_body))
                         {
-                            message.Content = new StringContent(_body);
+                            message.Content = new StringContent(_body, Encoding.UTF8, "application/json");
```

Comment:
@jcagme is it OK to make this json for call?

Reply:
Yes. We always serialize the objects been created/updated to JSON with ‘body = JsonConvert.SerializeObject’.